### PR TITLE
Fixes #179 and Fixes #180

### LINKED ISSUE #179: Add PlotTypes as array
and keep existing PlotType for now for compatibility reasons

### LINKED ISSUE #180: Make title and content optional in Section

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: schemas/OSP_Qualification_Plan_Schema.json [hunk 1]
```diff
@@ -114,10 +114,19 @@
             "$ref": "#/definitions/sections"
           }
         },
-        "required": [
-          "Id",
-          "Title",
-          "Content"
+        "anyOf": [
```
Comment (msevestre) [inline schemas/OSP_Qualification_Plan_Schema.json:117]: @Yuri05 This is how I managed to have either one or the other required

File: schemas/OSP_Qualification_Plan_Schema.json [hunk 2]
```diff
@@ -738,406 +757,442 @@
...
               }
             },
-            "required": [
-              "SectionId",
-              "Title",
-              "PlotType",
-              "Groups"
-            ],
-            "additionalProperties": false
+            "oneOf": [
```
Comment (msevestre) [inline schemas/OSP_Qualification_Plan_Schema.json:760]: one of here because we do not want PloType AND PlotTypes

**Review by msevestre — COMMENTED** [review]
indent was a bit off it seems. Changes are small however

File: schemas/OSP_Qualification_Plan_Schema.json (hunk 1 above)
Comment (Yuri05) [reply schemas/OSP_Qualification_Plan_Schema.json:117]: nice

**Pull request merged**